lib/fake_artist/table.ex

```diff
@@ -243,4 +283,37 @@ defmodule FakeArtist.Table do
       current_seat + 1
     end
   end
+
+  @spec everyone_has_voted(map()) :: boolean()
+  defp everyone_has_voted(state) do
+    game_master_vote = 1
+
+    Enum.count(state.players |> Map.values(), fn x -> x.voted_for == nil end) ==
+      0 + game_master_vote
```

Comment:
I think this would be clearer subtracted from the left side.
`x - y == 0` reads better to me than `x == 0 + y`

Reply:
alright